docs/contributing/development.md

```diff
@@ -0,0 +1,41 @@
+(dev)=
+# Development
+
+(devenv)=
+## Tools and environment
+
+This repository makes use of a number of tools. At a minimum, you'll want to
+install the following on your machine:
+
+- [Go](https://go.dev/dl/), for building and running the code in this
+  repository, including the Go SDK.
+- [NodeJS](https://nodejs.org/en/download/), for working with the NodeJS SDK.
+- [Python](https://www.python.org/downloads/), for working with the Python SDK.
+- [.NET](https://dotnet.microsoft.com/download), for working with the .Net SDK.
```

Comment:
Why this and not Java? Mise doesn't seem to install this either.  We should probably either mention both here (and install them using mise), or neither.

Reply:
For now I've just replicated the list from the `CONTRIBUTING.md`. I imagine it's because we still have `pkg/codegen/dotnet` in this repository, which isn't ideal, and that's why `dotnet` is needed in places. Not being in Mise is an oversight on my part -- it's not a core Mise thing, but I think there's a module we can use. Will investigate and raise separately. Good catch :pray: